### PR TITLE
JSDK-110 - As a SDK user i would like to avoid double posting manual journal entries via api

### PR DESCRIPTION
**Pull request by acostros**

### CODE REVIEW

File: src/com/mambu/apisdk/services/AccountingService.java [hunk 1]
```diff
@@ -234,23 +234,97 @@ public GLAccount getGLAccount(String glCode, String fromDate, String toDate) thr
...
+	 * @param params
+	 *            the parameters of the call
+	 * @return a list containing successful posted entries
+	 * 
+	 * @throws MambuApiException
+	 */
+	private List<GLJournalEntry> executePostGLJournalEntries(ParamsMap params) throws MambuApiException {
+
+		// Create ApiDefinition
+		ApiDefinition apiDefiinition = new ApiDefinition(APIData.GLJOURNALENTRIES, ContentType.WWW_FORM, Method.POST,
```
Comment (florinparascan) [inline src/com/mambu/apisdk/services/AccountingService.java]: apiDefiinition - I know that this was converted to a stand-alone method, but please correct the variable name

File: src/demo/DemoTestAccountingService.java [hunk 2]
```diff
@@ -132,35 +134,78 @@ public static void testPostGLJournalEntries(List<GLAccount> allAccounts) throws
...
+			System.out.println(
+					"\tID=" + entry.getEntryId() + "\tAmount=" + entry.getAmount() + "\tType=" + entry.getType());
+		}
+
+	}
+	
+	public static void testPostGLJournalEntriesWithTransactionChannel(List<GLAccount> allAccounts)
+			throws MambuApiException {
+
+		System.out.println("\nIn testPostGLJournalEntries");
```
Comment (florinparascan) [inline src/demo/DemoTestAccountingService.java]: please update the name

File: src/demo/DemoTestAccountingService.java [hunk 3]
```diff
@@ -132,35 +134,78 @@ public static void testPostGLJournalEntries(List<GLAccount> allAccounts) throws
...
 		for (GLJournalEntry entry : gLJournalEntries) {
-			System.out.println("\tID=" + entry.getEntryId() + "\tAmount=" + entry.getAmount() + "\tType="
-					+ entry.getType());
+			System.out.println(
+					"\tID=" + entry.getEntryId() + "\tAmount=" + entry.getAmount() + "\tType=" + entry.getType());
+		}
+
+	}
+	
+	public static void testPostGLJournalEntriesWithTransactionChannel(List<GLAccount> allAccounts)
```
Comment (florinparascan) [inline src/demo/DemoTestAccountingService.java]: channel? id?

**Comment by florinparascan** [comment]
Finished the review

File: src/demo/DemoTestAccountingService.java (hunk 3 above)
Comment (acostros) [inline src/demo/DemoTestAccountingService.java]: it is actually the Id, thanks!

File: src/com/mambu/apisdk/services/AccountingService.java (hunk 1 above)
Comment (acostros) [inline src/com/mambu/apisdk/services/AccountingService.java]: sure, fixed, thanks!

**Pull request merged**